text/_util.ts

```diff
@@ -18,3 +18,9 @@ export function splitToWords(input: string) {
 export function capitalizeWord(word: string): string {
   return word ? word[0]!.toUpperCase() + word.slice(1).toLowerCase() : word;
 }
+
+export function generateRandomString(min: number, max: number): string {
```

Comment:
Because this util is only used in benchmark, can you move this to somewhere else? (like `_test_util.ts`?)

Reply:
@timreichen sorry, but could you elaborate? i don't follow how shuffle could be used to generate random string.

Reply:
@kt3k applied and force-pushed in https://github.com/denoland/std/commit/ed38858d32ccae418dbff4a195e2c435c61f36f4